templates/overrides/navigation/oe-corporate-blocks-eu-footer.html.twig

```diff
@@ -54,14 +54,14 @@
 
   {# Follow section must always be the last on the left. #}
   {% set _section_2 = _section_2|merge([{
-    'title': 'Follow us'|t,
+    'title': 'Follow us on social media'|t,
```

Comment:
check which strings we have translations for in oe_theme to copy over, and report the ones we don't, so we can ask for them